### DISCUSSION TITLE
New combos are messed up by Ctrl+G

### DISCUSSION THREAD

**Discussion by cihe13375** [ppy/osu#27315] [Q&A] [answered]
Let's say I want to reverse the order of the yellow triangle:
![1](https://github.com/ppy/osu/assets/33959010/59574dc3-bdaa-4d33-b599-f989aa86d5c6)
If I press Ctrl+G the lazer editor gives this:
![3](https://github.com/ppy/osu/assets/33959010/4fe0a4b8-6639-4bb8-b128-78a7d3575b2d)
But really, in 99% of cases I'd to like to see this (and it is what stable editor does):
![2](https://github.com/ppy/osu/assets/33959010/b37b41f3-a404-4aca-a9a8-0bd719a88b78)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/27319
